edge-modules/edgehub-proxy/src/main.rs

```diff
@@ -148,17 +144,16 @@ fn run() -> Result<(), Error> {
         let common_name = args
             .value_of("common name")
             .expect("no value for required COMMON_NAME");
-        let expiration = args
-            .value_of("expiration")
-            .expect("no value for required EXPIRATION");
+        let expiration = DateTime::parse_from_rfc3339(
+            args
+                .value_of("expiration")
+                .expect("no value for required EXPIRATION"))?;
+        let expiration_utc = expiration.with_timezone(&Utc);
         info!("Retrieving server certificate with common name \"{}\" and expiration \"{}\" from {}...", common_name, expiration, url);
 
-        let cert_request =
-            ServerCertificateRequest::new(common_name.to_string(), expiration.to_string());
+
         let request =
-            client
-                .workload_api()
-                .create_server_certificate(api_version, module, gen, cert_request);
+            client.create_server_cert(module, gen, common_name, expiration_utc);
```

Comment:
When you're put `tokio::main` you should be able just do this:
```
let response = client.create_server_cert(module, gen, common_name, expiration_utc).await?;
```

Reply:
Yup. That worked perfect and looks much more readable. Thanks